modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -170,7 +175,11 @@ export default {
   },
   created() {
     this.options.filters.forEach(filter => {
-      this.filterValues[filter.name] = filter.def || filter.choices[0].value;
+      this.filterValues[filter.name] = filter.def;
```

Comment:
There might not be any choices yet. Better to require explicit defaults, or do the "first value as fallback default" dance more carefully.